frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/backgroundables/BaseSlideInBackgroundable.java

```diff
@@ -86,11 +88,14 @@ public void run(ProgressIndicator indicator) {
       return;
     }
 
-    Try.run(() -> branchLayoutWriter.write(macheteFilePath, newBranchLayout, /* backupOldLayout */ true))
-        .onFailure(t -> VcsNotifier.getInstance(project).notifyError(/* displayId */ null,
-            /* title */ getString(
-                "action.GitMachete.BaseSlideInBelowAction.notification.title.branch-layout-write-fail"),
-            getMessageOrEmpty(t)));
+    ApplicationManager.getApplication()
+        .invokeLater(() -> WriteAction
```

Comment:
is it needed to run `compute` within `invokeLater`?

Reply:
According to this:
<img width="601" alt="image" src="https://user-images.githubusercontent.com/61919032/197785613-df8a5af0-f676-4188-9654-56ec9a869fb3.png">
And this [[source]](https://plugins.jetbrains.com/docs/intellij/general-threading-rules.html#modality-and-invokelater):
<img width="729" alt="image" src="https://user-images.githubusercontent.com/61919032/197785893-bcfb45b0-95da-4737-a1cd-42f84765cb5f.png">

I think it should be inside `invokeLater` when we're using a background thread